x-pack/plugins/security/server/authentication/authenticator.ts

```diff
@@ -274,6 +275,7 @@ export class Authenticator {
           name: '__http__',
           logger: options.loggers.get(HTTPAuthenticationProvider.type),
           urls: {
+            // @ts-expect-error
```

Comment:
Nit: I personally prefer to add some text to this comment explaining what we're trying to ignore. At a minimum, I just copy the TS error verbatim if it's not too long